W-13222879-consoleuserspecificationneedsbotv12-se (#177)
mulesoft/docs-rpa: rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc

+
The bot can overtake the console or RDP session only if the specified user is logged in at runtime. If no user or another user is logged in at runtime, the process run results show the failure *No active session with the user '<user name>' could be found on the system*. 
+
[WARNING]

> Bots prior to version 1.2.0 overtake an open console session without checking if the logged-in user matches the specified user.

Reviewer: ```suggestion
Bots running an earlier version than 1.2.0 overtake an open console session without checking if the logged-in user matches the specified user.
```

+
The bot can overtake the console or RDP session only if the specified user is logged in at runtime. If no user or another user is logged in at runtime, the process run results show the failure *No active session with the user '<user name>' could be found on the system*. 
+
[WARNING]

> Bots prior to version 1.2.0 overtake an open console session without checking if the logged-in user matches the specified user.

Reviewer: ```suggestion
Bots running an earlier version than 1.2.0 overtake an open console session without checking if the logged-in user matches the specified user.
```
